apollo-normalized-cache-api/api/apollo-normalized-cache-api.api

```diff
@@ -164,6 +166,7 @@ public final class com/apollographql/apollo3/cache/normalized/api/Record : java/
 	public final fun getFields ()Ljava/util/Map;
 	public final fun getKey ()Ljava/lang/String;
 	public fun getKeys ()Ljava/util/Set;
+	public final fun getLastUpdated ()Ljava/util/Map;
```

Comment:
Same here. This is marked as ApolloExperiemental unfortunately, that's not taken into account here